integrationTests/package.json

```diff
@@ -0,0 +1,143 @@
+{
+  "name": "commerce-sdk-integration-test",
+  "version": "1.4.1-beta.0",
+  "description": "Salesforce Commerce Cloud SDK Integration Test",
+  "homepage": "https://developer.commercecloud.com/s/",
+  "private": true,
+  "scripts": {
+    "depcheck": "depcheck",
+    "lint": "eslint . --ext .ts,.js --quiet --fix",
+    "pretest": "npm run lint && npm run depcheck",
+    "test:ts": "nyc mocha \"tests/*.test.ts\"",
+    "test:js": "nyc mocha \"tests/*.test.js\"",
+    "test": "npm run test:ts && npm run test:js",
+    "redis": "npm run redis --prefix ../packages/core",
+    "redis:shutdown": "npm run redis:shutdown --prefix ../packages/core",
+    "redis:debug": "npm run redis:debug --prefix ../packages/core",
+    "redis:debug:shutdown": "npm run redis:debug:shutdown --prefix ../packages/core",
+    "bundle:core": "npm pack ../packages/core && mv commerce-apps-core*.tgz commerce-apps-core.tgz",
```

Comment:
Potential one off issue for bundle:core script...
Can u pls test the following and see if it works..

1. npm run bundle:core
2. Change the version in core package
3. npm run bundle:core